include/_kernel.h

```diff
@@ -2395,152 +2445,173 @@ __IF_FP64(_CL_DECLARE_PREFETCH_FUNCS(double));
 
 #endif
 
-
-
-/* read_imagef 2d functions*/
-float4 _CL_OVERLOADABLE read_imagef (IMG_RO_AQ image2d_t image, sampler_t sampler,
-                                     int2 coord);
-/* float coords not implemented yet
-float4 _CL_OVERLOADABLE read_imagef (IMG_RO_AQ image2d_t image, sampler_t sampler,
-                                     float2 coord);
-*/
+/* NO sampler */
 
 float4 _CL_OVERLOADABLE read_imagef (IMG_RO_AQ image2d_t image, int2 coord);
-
 float4 _CL_OVERLOADABLE read_imagef (IMG_RO_AQ image2d_array_t image, int4 coord);
+float4 _CL_OVERLOADABLE read_imagef (IMG_RO_AQ image3d_t image, int4 coord);
 
-float4 _CL_OVERLOADABLE read_imagef (IMG_RO_AQ image2d_array_t image, sampler_t sampler,
-                                     int4 coord);
+uint4 _CL_OVERLOADABLE read_imageui ( IMG_RO_AQ image2d_t image, int2 coord);
```

Comment:
Check the style. Extra whitespace after ( at least. And return value should be in a separate row per GNU C style.